loopingATriangle.js

```diff
@@ -0,0 +1,5 @@
+var ptrn = '#';
```

Comment:
generally speaking, bias towards using whole words for variable naming. there should also be an additional space between this line and the `for` loop.